centreon/www/front_src/src/Dashboards/Dashboards.tsx

```diff
@@ -1,13 +1,147 @@
+  deleteDialogStateAtom,
+  isDialogOpenAtom,
+  selectedDashboardAtom
+} from './atoms';
+import useSubmitDashboard from './useSubmitDashboard';
+import Listing from './Listing';
+import useRemoveDashboard from './useRemoveDashboard';
+
+import { ModalActionsLabels } from 'packages/ui/src/components/Modal/ModalActions';
+
+const Dashboards = (): ReactElement => {
+  const { t } = useTranslation();
+
+  const [deleteDialogState, setDeleteDialogState] = useAtom(
+    deleteDialogStateAtom
+  );
+  const isDialogOpen = useAtomValue(isDialogOpenAtom);
+  const selectedDashboard = useAtomValue(selectedDashboardAtom);
+  const closeDialog = useSetAtom(closeDialogAtom);
+
+  const { submit } = useSubmitDashboard();
+  const { remove: removeDashboard } = useRemoveDashboard();
+
+  const labels = useMemo(
+    (): {
+      deleteConfirmation: { actions: ModalActionsLabels; description: string };
+      form: DashboardFormLabels;
+      modalTitle: { create: string; delete: string; update: string };
+    } => ({
+      deleteConfirmation: {
+        actions: {
+          cancel: t(labelCancel),
+          confirm: t(labelDelete)
+        },
+        description: t(labelDescriptionDeleteDashboard)
+      },
+      form: {
+        actions: {
+          cancel: t(labelCancel),
+          submit: {
+            create: t(labelCreate),
+            update: t(labelUpdate)
+          }
+        },
+        entity: {
+          description: t(labelDescription),
+          name: t(labelName)
+        }
+      },
+      modalTitle: {
+        create: t(labelCreateDashboard),
+        delete: t(labelDeleteDashboard),
+        update: t(labelUpdateDashboard)
+      }
+    }),
+    []
+  );
+
+  return (
+    <TiledListingPage>
+      <Header title={t(labelDashboards)} />
+      <Suspense fallback={<ListingSkeleton />}>
+        <Listing />
+      </Suspense>
+      <Modal open={isDialogOpen} onClose={closeDialog}>
+        <Modal.Header>
+          {labels.modalTitle[selectedDashboard?.variant ?? 'create']}
+        </Modal.Header>
+        <Modal.Body>
+          <DashboardForm
+            labels={labels.form}
+            resource={
+              (selectedDashboard?.dashboard as DashboardResource) || undefined
+            }
+            variant={selectedDashboard?.variant}
+            onCancel={closeDialog}
+            onSubmit={submit}
+          />
+        </Modal.Body>
+      </Modal>
+      <Modal
+        open={deleteDialogState.open}
+        onClose={() =>
+          setDeleteDialogState({
+            ...deleteDialogState,
+            open: false
+          })
+        }
+      >
+        <Modal.Header>{labels.modalTitle.delete}</Modal.Header>
+        <Modal.Body>
+          <p
+            /* eslint-disable-next-line react/no-danger */
+            dangerouslySetInnerHTML={{
+              __html:
+                t(labelDescriptionDeleteDashboard, {
+                  name: deleteDialogState.item?.name
+                }) || ''
+            }}
```

Comment:
This is kind of dangerous. why not just `<p>{t(labelDescriptionDeleteDashboard, {
                  name: deleteDialogState.item?.name
                })}</p>`

Reply:
because the `name` value has to be styled with `<strong>` tag, so there's no other option afaik.
**→ note that the actual `name` value is being escaped by `t()`**

```
const labelDescriptionDeleteDashboard = 'Are you sure you want to delete <strong>{{name}}</strong> ?';

<p
  /* eslint-disable-next-line react/no-danger */
  dangerouslySetInnerHTML={{
    __html:
      t(labelDescriptionDeleteDashboard, {
        name: deleteDialogState.item?.name
      }) || ''
  }}
/>  
```

Reply:
I'm bringing @sc979 here. We might want to mitigate by not having the name displayed in bold.

Reply:
Or we split the sentence to extract the translated string from it

Reply:
Yes, the best is to split the sentence

Reply:
ok, let's split for the time being

Reply:
replaced with 
`            {t(labelDescriptionDeleteDashboardPartOne)}
            <strong>{deleteDialogState.item?.name}</strong>
            {t(labelDescriptionDeleteDashboardPartTwo)}`
Thank you @g10